source/assets/javascripts/all.js

```diff
@@ -3,9 +3,15 @@
 //= require 'lib/_masonry.pkgd'
 //= require 'lib/_handlebars-v4.0.5'
 //= require 'lib/_ramda'
+//= require 'lib/_raphael-min.js'
+//= require 'lib/_jquery.mapael.js'
+//= require 'lib/_world_countries.js'
+//= require 'lib/_jquery.mousewheel.min.js'
```

Comment:
This is going down a rabbithole if we're going to add all JS and CSS in all pages for something that is only needed in a single page and loading up all pages to make them render slower.

Can you make it a separate JS file and include it in `layout.erb` —

```
<%= auto_stylesheet_link_tag %>
<%= auto_javascript_link_tag %>
```